docs/limitations/nested-objects.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[nested-objects]]
+== Limitation with nested objects
+
+Kibana is not able to perform aggregations across fields containing nested
+documents.
+
+[float]
+=== Workaround
+
+For simple aggregations (e.g. a `terms` aggregation), you
+may consider adding the `include_in_parent` setting to your Elasticsearch
+mapping. This creates a flattened version of a field in the nested document at
+the parent level.
+
+[source,js]
+----------------------------------
+{
+  "type1": {
+    "properties": {
+      "users": {
+        "type": "nested",
+        "include_in_parent": true,
+        "properties": {
+          "first": { "type": "keyword" },
+          "last": { "type": "keyword" }
+        }
+      }
+    }
+  }
+}
+----------------------------------
+
+[IMPORTANT]
+==============================================
+The `include_in_parent` workaround mentioned above is not a supported
+workaround. Please use it at your own risk.
+==============================================
```

Comment:
I will suggest not mentioning the workaround at all in documentation (leave it in the public GH for an open discussion) given that Elasticsearch is already planning on deprecating include_in_parent, and possibly disabling it in 5.4.0+.